analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -164,6 +169,14 @@ class Gradle(
             gradleConnector.daemonMaxIdleTime(10, TimeUnit.SECONDS)
         }
 
+        // Gradle's default heap value is 512 MiB which is to low for bigger projects.
+        // See https://docs.gradle.org/current/userguide/build_environment.html#sec:configuring_jvm_memory.
+        // Setting the default to 8 GiB if no value is set in ~/.gradle/gradle.properties.
+        val maxHeapArg = gradleProperties.firstOrNull { (key, value) ->
+            key.startsWith("org.gradle.jvmargs")
+                .and(value.contains(other = "-xmx", ignoreCase = true))
```

Comment:
I believe this could simply be `key == "org.gradle.jvmargs" && value.contains("-xmx", ignoreCase = true)`.

Reply:
But thinking about it, the current implementation would override *any* set `org.gradle.jvmargs` which do *not* contain "-xmx". So how about a slightly different implementation that just "forwards" `org.gradle.jvmargs` but appends "-Xmx8g" to it if it does not already contain "-xmx"?